src/main/java/duke/FindCommand.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class FindCommand extends Command {
+    private String keyword;
+    
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) throws DukeException, IOException {
+        ArrayList<Task> list = new ArrayList<>();
```

Comment:
Perhaps it would be better if you use a more appropriate name instead of list? Such as tasks?

Reply:
Alright thanks